### ISSUE TITLE
Add support for more languages

### ISSUE THREAD

**Issue by kristibektashi** [leecher1337/ntvdmx64#117]

**Comment by revelator** [comment]
seconded :)

**Comment by leecher1337** [comment]
You want translated WOW16 (as there currently aren't any language resources shipped with it)?
Most of the DOS stuff is already available in different languages.

**Comment by kristibektashi** [comment]
> You want translated WOW16 (as there currently aren't any language resources shipped with it)?
> Most of the DOS stuff is already available in different languages.

Yes, it would be nice to have language support in WOW16. Also can you add support for the DOS stuff in Greek?

**Comment by kristibektashi** [comment]
@leecher1337

**Comment by dominicraf** [comment]
Where can I find a list of supported languages (for setting LANG=)? Actually I just want to build in English, previously I used LANG=usa but this no longer works ('Language usa is invalid, aborting'). I have also tried us and en without success.

**Comment by leecher1337** [comment]
List of languages is in https://github.com/leecher1337/ntvdmx64/blob/master/ntvdmpatch/doc/minnt.txt
Just tested it with current autobuild

`set LANG=usa
autobuild-ccpu-chk.cmd 
`

on 64bit cmd still works fine

**Comment by dominicraf** [comment]
Strange. I was using it in a Powershell window, running as administrator (Windows 11, 64bit). I will try it again next time I have to rebuild. Actually my batch file reads:
```
set LANG=usa
call autobuild-ccpu-fre.cmd

```

**Comment by leecher1337** [comment]
I just built as a normal user, no Admin elevation (WIN+R / cmd / OK) in Windows 7 x64.
